### PR TITLE
feat: add background support on DIV and A elements.

### PR DESCRIPTION
**Pull request by euskadi31**

### CODE REVIEW

**Comment by vvo** [comment]
Hi! Thanks for the contribution.

Once you are done (maybe already the case?), can you:
- provide some context (what are you trying to do? Is there already a linked issue? Maybe we can make this a plugin or a recipe, should this be in core?)
- document the API in the github issue (we will then see if it's a good fit, how do we make it to the readme)
- remove the built file from the commit (I will build it when releasing if necessary)

**Comment by euskadi31** [comment]
I added a example for background lazyload feature.

File: test/background.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    assert.equal(realSrc, test.getAttribute('src'));
+  });
+
+  describe('after some scrolling', function() {
+    beforeEach(h.scroller(0, 50));
+    beforeEach(h.wait(50));
+    beforeEach(h.scroller(0, 0));
+    beforeEach(h.wait(50));
+
+    it('loads the image when visible for a while', h.eltLoaded('simple_background'));
```
Comment (vvo) [inline test/background.js]: I guess here what you need to check in the test is that the style property was updated with the good background. Can you do that?

**Comment by vvo** [comment]
@euskadi31 Thats very nice, can you just update the test according to my comment?

**Comment by vvo** [comment]
For this PR, we still need:
- [ ] good documentatino
- [ ] remove builds from commits

I can do that, no problem, if you want to do it that's good also. Thanks a lot for contribution

File: test/background.js (hunk 1 above)
Comment (euskadi31) [inline test/background.js]: yep ;)

**Pull request closed without merging**